src/retryRequests.ts

```diff
@@ -76,7 +76,7 @@ export function addRetryToAxiosInstance(instance: AxiosInstance) {
       }
 
       if (!err.response) {
-        return true;
+        return false;
```

Comment:
what does this do? won't retry if there is a falsy response?

Reply:
Just saying that it shouldn't retry the request if we didn't get a response from the API (we don't know the status code or the request failed)